src/basic/sysctl-util.c

```diff
@@ -56,12 +76,18 @@ int sysctl_write(const char *property, const char *value) {
         if (!path_is_normalized(p))
                 return -EINVAL;
 
+        ret = sysctl_f_ofd_lk(p, lock);
+        if (ret < 0) {
+                log_warning("Skipping sysctl write to '%s', it's handled elsewhere.", p);
+                return -EBUSY;
```

Comment:
merge these two lines into `return log_warning_errno(r, "Skipping …");`

And drop the {} then